Description: tweak lcdproc config for debian
 patch LCDd.conf to:
 * use syslog instead of stderr to show message
 * run LCDd as root (to read /dev/lcd* file)
 .
 The latter could be done better by tweaking udev rule
Author: dod
Applied-Upstream: NA
--- a/LCDd.conf
+++ b/LCDd.conf
@@ -62,11 +62,11 @@
 #ReportLevel=3
 
 # Should we report to syslog instead of stderr ? [default: no; legal: yes, no]
-#ReportToSyslog=yes
+ReportToSyslog=yes
 
 # User to run as.  LCDd will drop its root privileges, if any,
 # and run as this user instead.
-User=nobody
+# User=nobody
 
 # The server will stay in the foreground if set to true.
 #Foreground=no
